typescript/sdk/src/token/deploy.ts

```diff
@@ -135,50 +137,63 @@ export class HypERC20Deployer extends GasRouterDeployer<
     chain: ChainName,
     config: HypERC20CollateralConfig,
   ): Promise<HypERC20Collateral> {
-    let contractName:
+    let tokenType:
       | TokenType.fastCollateral
       | TokenType.collateral
       | TokenType.collateralVault;
     switch (config.type) {
       case TokenType.fastSynthetic || TokenType.fastCollateral:
-        contractName = TokenType.fastCollateral;
+        tokenType = TokenType.fastCollateral;
         break;
       case TokenType.collateral:
-        contractName = TokenType.collateral;
+        tokenType = TokenType.collateral;
         break;
       case TokenType.collateralVault:
-        contractName = TokenType.collateralVault;
+        tokenType = TokenType.collateralVault;
         break;
       default:
```

Comment:
isnt config.type already `tokenType`?

Reply:
yeah but if type is fastSynthetic, it deploys fastCollateral
```
      case TokenType.fastSynthetic || TokenType.fastCollateral:
        tokenType = TokenType.fastCollateral;
```